infra/scripts/go-deps.sh

```diff
@@ -21,19 +21,32 @@
 #     pkg       This is github.com/vmware/vic/cmd/imagec for example
 #
 
+cache_dir=.godeps_cache
+
 pkg=$1
 flags=$2
+cachedname=`echo .$1.godeps_cache | sed 's/\//_/g'`
 
 if [ -d "$pkg" ]; then
-    if [[ "$flags" == *d* ]]
-    then
+
+    if [[ "$flags" == *d* ]]; then
         # Only output if make is given the '-d' flag
         echo "Generating deps for $pkg" >&2
     fi
 
-    go list -f '{{join .Deps "\n"}}' github.com/vmware/vic/"$pkg" 2>/dev/null | \
-        xargs go list -f '{{if not .Standard}}{{.ImportPath}}{{end}}' 2>/dev/null | \
-        sed -e 's:github.com/vmware/vic/\(.*\)$:\1/*:'
+    if [ -n "$VIC_CACHE_DEPS" ]; then
```

Comment:
Can we note the `export VIC_CACHE_DEPS=1` requirement somewhere in this file? It's only mentioned in the commit message, so having it noted here will help.